docker-compose.yml

```diff
@@ -3,8 +3,8 @@ services:
   nginx:
     image: bitnami/nginx
     volumes:
-      - /home/debian/ircd2/nginx/conf/private/server_block.conf:/opt/bitnami/nginx/conf/server_blocks/my_server_block.conf:ro
-      - /home/debian/ircd2/certs:/certs
+      - $HOME/irc.wetfish.net/nginx/conf/private/server_block.conf:/opt/bitnami/nginx/conf/server_blocks/my_server_block.conf:ro
```

Comment:
It is unnecessary to use `$HOME/irc.wetfish.net`. You should simply use a path relative to the compose file here. 

As per the Docker volume docs: https://docs.docker.com/compose/compose-file/#volumes

```
  # Path on the host, relative to the Compose file
  - ./cache:/tmp/cache
```

Reply:
I've made the change to relative paths. Thank you for the doc reference. :+1: